ocw/lib/db.py

```diff
@@ -100,7 +103,8 @@ def ec2_to_local_instance(instance, vault_namespace, region):
         instance_id=instance.instance_id,
         state=StateChoice.ACTIVE,
         region=region,
-        csp_info=json.dumps(csp_info, ensure_ascii=False)
+        csp_info=json.dumps(csp_info, ensure_ascii=False),
+        max_age=timedelta(seconds=int(csp_info['tags'].get('openqa_ttl', 0))),
```

Comment:
why tag name is openqa_ttl and field name is max_age ? why we need to keep it inside csp_info and as a separate field at the same time ?

Reply:
`max_age` and `ttl` good hint, lets make it equal. What would you prefer?

csp_info is a JSON data string and it is hart to use in `WHERE` statements via SQL.
Actually, I would like to make all "real, mandatory data" into db fields. 
The next thing we should do is "Tags". WDYT?

Reply:
`ttl` is more accurate.

Reply:
> csp_info is a JSON data string and it is hart to use in `WHERE` statements via SQL.

yeah sure but my idea was that we could remove it from JSON because we are generating it . But after checking code I see that it might be overkill because we have generic logic which blindly copying all tags ...

Reply:
done